### PR TITLE
PLAN-558 remove some RRKs from Explore

### PR DESCRIPTION
**Pull request by lastminutediorama**
Not sure if this is the correct way to do this, but it seemed straightforward enough

### CODE REVIEW

**Comment by pflopez** [comment]
@lastminutediorama can you generate a scenario and double check if we get the layers correctly on the results (for removed rrks)
Screen Shot 2023-11-08 at 12 03 58

**Comment by pflopez** [comment]
Wondering if instead of removing it completely we'll need to turn `  "display": false,`

**Comment by lastminutediorama** [comment]
> Wondering if instead of removing it completely we'll need to turn ` "display": false,`

I'll have a look...

**Comment by lastminutediorama** [comment]
> Wondering if instead of removing it completely we'll need to turn ` "display": false,`

Well, changing `display: false` appears to do nothing.... 😬 I'll try a couple other options

**Comment by lastminutediorama** [comment]
> @lastminutediorama can you generate a scenario and double check if we get the layers correctly on the results ...


Welp, if we just remove the metrics from the `conditions.json` this _does_ cause them to also be removed from the results. So...if that's not what we want 😄 , we might have to make some changes...

Both the Explore control panel and the Scenario Results component seem to get their data more-or-less directly from the /conditions backend, (in the [config](https://github.com/OurPlanscape/Planscape/blob/6b511e8d099bc0d1bc9c36c163d0cc6557624603/src/planscape/conditions/views.py#L53) and [metrics](https://github.com/OurPlanscape/Planscape/blob/6b511e8d099bc0d1bc9c36c163d0cc6557624603/src/planscape/conditions/views.py#L74) view functions, respectively), without much transformation.

Also, it seems the "display" boolean appears to be used by the UI only for Pillars and Elements, but not for individual Metrics. 
For example, [where it builds the condition tree](https://github.com/OurPlanscape/Planscape/blob/6b511e8d099bc0d1bc9c36c163d0cc6557624603/src/interface/src/app/map/map-control-panel/condition-tree/condition-tree.component.ts#L217)   for the Map Control Panel in Explore.

 Note that there is a `display` attribute defined for Pillars and Element types, but there is [no `display?` attribute for Metrics](https://github.com/OurPlanscape/Planscape/blob/6b511e8d099bc0d1bc9c36c163d0cc6557624603/src/interface/src/app/types/data.types.ts#L51).  

If we want to just be able to hide some metrics from the Explore layer, I'd propose we change those data types and the filters in the UI, so we could include a "display" option for metrics in the `conditions.json` file. Thoughts?

File: src/planscape/config/conditions.json [hunk 1]
```diff
@@ -2295,44 +2282,6 @@
               "normalized_data_download_path": "sierra-nevada/airQualityTranslate/partMatter.tif",
               "display_name": "Particulate Matter",
               "metrics": [
-                {
```
[the remark this answers is not in the document]
Comment (lastminutediorama) [reply src/planscape/config/conditions.json:2298]: I'm actually not sure, it was a ticket assigned to me without much detail, so maybe it's best to pause on this until people get back from vacation.

**Pull request closed without merging**